README.md

```diff
@@ -66,53 +70,53 @@ wrappers over common tasks).
 
 ```lua
 require'telescope.builtin'.git_files{
-    -- See Picker for additional options
-    show_preview       = true, -- Show preview
-    prompt             = "Git File",
-    selection_strategy = "reset" -- follow, reset, line
+  -- See Picker for additional options
+  show_preview       = true, -- Show preview
+  prompt             = "Git File",
+  selection_strategy = "reset" -- follow, reset, line
 }
 ```
 
 ```lua
 require'telescope.builtin'.live_grep{
-    -- See Picker for additional options
-    prompt = "Live Grep",
+  -- See Picker for additional options
+  prompt = "Live Grep",
 }
 ```
 
 ```lua
 require'telescope.builtin'.lsp_references{
-    -- See Picker for additional options
-    prompt = 'LSP References'
+  -- See Picker for additional options
+  prompt = 'LSP References'
 }
 ```
 
 ```lua
 require'telescope.builtin'.quickfix{
-    -- See Picker for additional options
-    prompt = 'Quickfix'
+  -- See Picker for additional options
+  prompt = 'Quickfix'
```

Comment:
I don't think we need to put `prompt` in these -- it is auto supplied by the builtin. I think it makes people think they have to include it.

Reply:
This is fair but I wanted to show the API for each builtin and each one only uses prompt right now. 

Showing these, or any options, helps those who don't know lua syntax. Maybe this is outside of the scope of this project but making shown in syntax can help show others how lua will allow options. 

So ultimately do we want to show no options for these? The API will change some so maybe we can re-approach it after the global level is setup.